deep_translator/base.py

```diff
@@ -181,3 +202,14 @@ def _translate_batch(self, batch: List[str], **kwargs) -> List[str]:
             translated = self.translate(text, **kwargs)
             arr.append(translated)
         return arr
+
+    async def async_translate_batch(
+        self, batch: List[str], **kwargs
+    ) -> List[str]:
+        if not batch:
+            raise Exception("Enter your text list that you want to translate")
```

Comment:
Can you add this custom exception to exceptions.py just to keep everything consistent? Something like a NotValidInputBatch exception